src/shims/lock.h

```diff
@@ -526,22 +526,22 @@ typedef struct dispatch_once_gate_s {
  *
  * |     Thread A     |     Thread B     |     Thread C     |
  * |                  |                  |                  |
- * |   G ∈ { 0, 1 }   |                  |                  |
+ * |   G �? { 0, 1 }   |                  |                  |
```

Comment:
It looks like some Unicode symbols got mangled here

Reply:
Thank you!